cmd/ci-operator-prowgen/main.go

```diff
@@ -673,7 +683,11 @@ func generateJobs(
 		}
 
 		if element.Cron == nil {
-			presubmits[orgrepo] = append(presubmits[orgrepo], *generatePresubmitForTest(element.As, info, label, podSpec, true, configSpec.CanonicalGoRepository))
+			presubmit := *generatePresubmitForTest(element.As, info, label, podSpec, true, configSpec.CanonicalGoRepository)
```

Comment:
nit: don't dereference this pointer here if you're just going to reference it again two lines down

Reply:
Duplicating jobs will be with python and wont be on our code base.